content/en/blog/_posts/2024-04-02-introducing-jobset/2024-04-02-introducing-jobset.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "Introducing JobSet"
+date: 2024-04-02
+slug: introducing-jobset
+---
+
+**Authors**: Daniel Vega-Myhre (Google), Abdullah Gharaibeh (Google), Kevin Hannon (Red Hat)
+
+In this article, we introduce [JobSet](https://jobset.sigs.k8s.io/), an open source API for representing distributed jobs. The goal of JobSet is to provide a stable API for running/building APIs with AI/ML and HPC use cases in mind.
```

Comment:
I struggle if we should define HPC..

What do you mean by "running/building APIs"?

Reply:
I think HPC is a known term, but I also don't understand the running/building APIs part.